Readme.md

```diff
@@ -342,6 +342,8 @@ cp config.ini-sample ~/.elodie/config.ini
 # now you're ready to add your MapQuest key
 ```
 
+You may also prefer your locations to be named in English language, not the local language. Then just set the following key in `config.ini` like so: `prefer_english_names=True`.
```

Comment:
Mind updating this to...

```
If you're an english speaker then you will probably want to add `prefer_english_names=True` to the `[MapQuest]` section else you'll have cities named using the local language.
```

Reply:
Well, this key will be in config by default, because it is now in sample config. Do you thing it should be enabled by default? I mean, if person can install and use Elodie he or she is certainly an English speaker.

Reply:
I think what you've done by defaulting it to `False` in the code and sample config is correct. While most people may prefer to have english names we should avoid making any changes which are not backward compatible. I wouldn't want someone without the config value to update their code and see different behavior.

Reply:
That's the point I completely agree with you. Good tone would be to possibly make this change in major version 2 or even better and a notice to the script about the setting and possible side effects.